ast/Trees.cc

```diff
@@ -907,6 +907,17 @@ string Send::toStringWithTabs(const core::GlobalState &gs, int tabs) const {
 string Send::showRaw(const core::GlobalState &gs, int tabs) {
     fmt::memory_buffer buf;
     fmt::format_to(std::back_inserter(buf), "{}{{\n", nodeName());
+
+    auto stringifiedFlags = vector<string>{};
+    if (this->flags.isPrivateOk) {
+        stringifiedFlags.emplace_back("privateOk");
+    }
+    if (this->flags.isRewriterSynthesized) {
+        stringifiedFlags.emplace_back("rewriter");
```

Comment:
if someone sees "rewriter" in the raw output and they're new to the codebase, it might take them a bit to know that it means they should look up the `isRewriterSynthesized` flag. maybe we should print out `rewriterSynthesized`? it's not like we're already trying to be concise in the raw output.

Reply:
That's a reasonable suggestion, I'll make that change.